src/cisco_gnmi/__init__.py

```diff
@@ -30,4 +30,4 @@
 from .xe import XEClient
 from .builder import ClientBuilder
 
-__version__ = "1.0.10"
+__version__ = "1.0.11"
```

Comment:
Thought we are at "1.0.11" now so should this be bumped to "12"?

Reply:
Merging into another branch for collective bump.